.gitignore

```diff
@@ -2,3 +2,4 @@ target
 logs
 project/project
 .bsp
+.idea/
```

Comment:
Would rather have it in `.git/info/exclude` as ide specific

Reply:
Removed.